conversions/base_changer.py

```diff
@@ -0,0 +1,104 @@
+num = input("Enter the number ").strip()
```

Comment:
This line will halt our automated tests.

Reply:
Added __main__